src/i18n/en.json

```
@@ -124,16 +126,21 @@
   "components.customers.CustomerInfo.customerHasHitasOwnership": "Customer has Hitas ownership",
   "components.customers.CustomerInfo.customerIsOver55": "Customer is over 55 years old",
   "components.customers.CustomerInfo.dateOfBirth": "Date of birth",
+  "components.customers.CustomerInfo.email": "Email",
   "components.customers.CustomerInfo.errorNoCustomer": "Error, no customer data",
   "components.customers.CustomerInfo.extraInfo": "Additional Info",
   "components.customers.CustomerInfo.familyWithChildren": "Family with Children",
   "components.customers.CustomerInfo.haso": "HASO",
   "components.customers.CustomerInfo.hasoNumber": "Aso number",
   "components.customers.CustomerInfo.hitas": "Hitas",
   "components.customers.CustomerInfo.lastContactDate": "Last Contact Date",
+  "components.customers.CustomerInfo.latestApplicantTab": "Latest Applicant Info",
   "components.customers.CustomerInfo.name": "Name",
   "components.customers.CustomerInfo.nin": "National Identification Number",
+  "components.customers.CustomerInfo.phoneNumber": "Phone number",
+  "components.customers.CustomerInfo.postalCode": "Postal code",
   "components.customers.CustomerInfo.registered": "Registered",
+  "components.customers.CustomerInfo.streetAddress": "Street address",
   "components.customers.CustomerSearchForm.btnSearch": "Search",
   "components.customers.CustomerSearchForm.email": "Email",
   "components.customers.CustomerSearchForm.firstName": "Fistname",
```

Not in the scope of the PR, but can you fix this, it irritates me :joy: 

```suggestion
  "components.customers.CustomerSearchForm.firstName": "First name",
```

---

There's also "Lastname" below it, ought to be "Last name"

---

Fistname was wild, fixed